recipes/cuda-compat/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set name = "cuda-compat" %}
+{% set version = "525.60.13" %}
+{% set cuda_version = "12.0" %}
```

Comment:
Is this the true cuda-compat versioning scheme / does this need to be a longer version like `12.0.0`? If cuda-compat can release new versions with CUDA updates for the same minor version, we need to be able to distinguish them.

Maybe we just need to use a longer version for the package version, and truncate it to `major.minor` for the `.so.{{ version }}` cases below?

Reply:
I can expand to the full CUDA version, we do bump driver versions in update releases.

This shouldn't need to be truncated for the below `.so`s as those are using the driver version.